releases/stellar/2025-01-GMP-v1.0.0.md

```diff
@@ -23,41 +23,71 @@ Changes in the release:
 
 ## Deployment
 
-Create an `.env` config.
+Create an `.env` config. `CHAIN` should be set to `stellar` for mainnet, and `stellar-2024-q4` for all other networks.
 
 ```yaml
 PRIVATE_KEY=xyz
 ENV=xyz
-CHAIN=stellar
+CHAIN=xyz
+```
+
+An initial Stellar chain config needs to be added to `${ENV}.json` file under `chains` key.
+
+```json
+"stellar": {
+    "name": "Stellar",
+    "axelarId": "stellar",
+    "rpc": "https://soroban-testnet.stellar.org",
+    "horizonRpc": "https://horizon-testnet.stellar.org",
+    "networkType": "testnet",
+    "chainType": "stellar",
+    "decimals": 7,
+    "finality": "1",
+    "approxFinalityWaitTime": 1,
+    "tokenSymbol": "XLM",
+    "tokenAddress": "CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC",
+    "explorer": {
+      "name": "Soroban Nownodes",
+      "url": "https://soroban.nownodes.io/testnet"
+    },
+  "contracts": {}
+  }
 ```
 
 Deploy Stellar Gateway
 
+1. Deploy Gateway
+
 ```bash
-# Deploy Gateway
-[1] node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0
+node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0
 ```
 
-[2] After deploying the Stellar Gateway, ensure that you [deploy CosmWasm for Stellar GMP v1.0.0](../cosmwasm/2025-01-STELLAR-GMP-v1.0.0.md).
+2. After deploying the Stellar Gateway, ensure that you [deploy CosmWasm for Stellar GMP v1.0.0](../cosmwasm/2025-01-Stellar-GMP-v1.0.0.md).
 
 - Once all steps are completed, update ampd with the Stellar chain configuration, then follow the remaining steps.
 
+3. Rotate genesis verifier set on Stellar Gateway
+
 ```bash
-# Rotate genesis verifier set on Stellar Gateway
-[3] node stellar/gateway.js rotate --chain-name $CHAIN
+node stellar/gateway.js rotate --chain-name $CHAIN
+```
+
+4. Deploy Gas Service
```

Comment:
Ordering doesn't quite make sense. All contracts can be deployed first before cosmwasm setup. This is more efficient since cosmwasm step can take multiple days on mainnet due to gov proposals